spec/features/user_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe 'User', type: :feature do
```

Comment:
Required

Please test the other features of the app as well, such as creating categories and expenses.